packages/cli/src/commands/composite/extract-model.ts

```diff
@@ -0,0 +1,58 @@
+import { Command, type CommandFlags } from '../../command.js'
+import { Flags } from '@oclif/core'
+import { readEncodedComposite, writeEncodedComposite } from '@glazed/devtools-node'
+
+type Flags = CommandFlags & {
+  output?: string
+}
+
+export default class CompositeExtractModel extends Command<Flags> {
+  static strict = false
+
+  static description = 'create a Composite from a list of model streams'
+
+  static flags = {
+    ...Command.flags,
+    output: Flags.string({
+      char: 'o',
+      description: 'path to the file where the composite representation should be saved',
+    }),
+  }
+
+  async run(): Promise<void> {
+    const parsed = await this.parse(CompositeExtractModel)
+    const allArgs = parsed.raw
+      .filter((token) => {
+        return token.type === 'arg'
+      })
+      .map((token) => {
+        return token.input
+      })
+
+    if (allArgs.length < 2) {
+      this.spinner.fail('Missing composite path and at least one model to extract')
+      return
+    }
+    try {
+      const compositePath = allArgs[0]
+      const modelsToExtract = allArgs.splice(1)
```

Comment:
Not an issue but this could simply be:
```suggestion
      const [compositePath, ...modelsToExtract] = allArgs
```